packages/grid/src/key-handlers/SelectionKeyHandler.ts

```diff
@@ -177,6 +177,13 @@ class SelectionKeyHandler extends KeyHandler {
 
       const { theme } = grid.props;
       const { autoSelectRow = false, autoSelectColumn = false } = theme;
+      const stickyOptions: StickyOptions = {
+        shouldStickBottom:
+          event.key === 'ArrowDown' ||
+          event.key === 'End' ||
+          event.key === 'PageDown',
+        shouldStickRight: event.key === 'ArrowRight',
+      };
```

Comment:
Similar to my other comment - we shouldn't be checking the key itself again, just look at the `deltaColumn`/`deltaRow` to set the sticky options. Also need to pass it through in the above `grid.moveCursorToPosition` call I think.

Reply:
Ah yes I missed that one just the very first moveCursorToPosition the other ones didn't need it